gcloud-java-core/src/main/java/com/google/gcloud/ServiceOptions.java

```diff
@@ -53,6 +53,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+@SuppressWarnings("rawtypes")
```

Comment:
Good point, I changed it so that I don't suppress warnings on the entire file.  `Service` was the only problem originally, and I added a type to it now.